aiven_db_migrate/migrate/pgmigrate.py

```diff
@@ -348,6 +348,10 @@ def pg_roles(self) -> Dict[str, PGRole]:
     def replication_available(self) -> bool:
         return self.version >= "10"
 
+    @property
+    def taken_replication_slots(self) -> int:
```

Comment:
I don't understand this property name. Could we come up with some clearer name?

Reply:
That's how we call a variable somewhere else in our code base. Basically what this returns is the number of replication slots that are already taken/occupied (I thought about "unavailable", "occupied", "used" and "taken" and at I went with "taken"), but I am open to your suggestions!

Reply:
> The `pg_replication_slots` view provides a listing of all replication slots that currently exist on the database cluster, along with their current state.

https://www.postgresql.org/docs/current/view-pg-replication-slots.html

So, how about `replication_slots_count` (concise) or `existing_replication_slots_count`? Also possible `num_replication_slots` or `total_replication_slots` (that one is a bit less clear that it's a number).

Yeah, while re-reading, I noticed that the name indicates a container of some sort, when we actually have an integer 😅. So I added `_count`.

Reply:
:heavy_check_mark: